src/test/java/org/apache/maven/plugin/resources/remote/RemoteResourcesMojoTest.java

```diff
@@ -51,6 +52,8 @@
 import org.eclipse.aether.internal.impl.SimpleLocalRepositoryManagerFactory;
 import org.eclipse.aether.repository.LocalRepository;
 
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
These days we're using Hamcrest, not assertj

Reply:
It is only test dependencies  .... I will not require 
even more JUnit 5 doesn't use it

Reply:
I'd like to settle on one thing. I don't care much which one, but let's not have a random mix of libraries doing the same thing.

Reply:
My impression is that across Maven and even more broadly across Java, but definitely within Maven hamcrest is far more common now. I've certainly seen it a lot more while working my way through all the plugins than I've seen assert4j, though they are both present. JUnit 4 already depends on Hamcrest, though that got removed in JUnit 5.

Reply:
ok we have Hamcrest so I would like to merge